src/ai/backend/manager/server.py

```diff
@@ -1041,10 +1074,39 @@ async def _call_cleanup_context_shutdown_handlers(app: web.Application) -> None:
     return app
 
 
-def build_internal_app() -> web.Application:
+def build_prometheus_service_discovery_handler(
+    root_ctx: RootContext,
+) -> Handler:
+    async def _handler(request: web.Request) -> web.Response:
+        services = await root_ctx.service_discovery.discover()
```

Comment:
Is this to pass `root_ctx` when the webapp is built not through a request context?